deployment/terraform/storage.tf

```diff
@@ -17,7 +17,29 @@ resource "aws_s3_bucket" "static" {
 
 resource "aws_s3_bucket" "storage" {
   bucket = "${lower("${var.environment}")}-pfb-storage-${var.aws_region}"
-  acl    = "private"
+  acl    = "public-read"
+  policy = <<POLICY
```

Comment:
Ok, set it up with the policy defined in `iam.tf` with the bucket name composed as above and attached in the bucket definition.

plan now says:
```
~ aws_s3_bucket.storage
    acl:                           "private" => "public-read"
    cors_rule.#:                   "0" => "1"
    cors_rule.0.allowed_headers.#: "0" => "1"
    cors_rule.0.allowed_headers.0: "" => "Authorization"
    cors_rule.0.allowed_methods.#: "0" => "1"
    cors_rule.0.allowed_methods.0: "" => "GET"
    cors_rule.0.allowed_origins.#: "0" => "1"
    cors_rule.0.allowed_origins.0: "" => "*"
    cors_rule.0.expose_headers.#:  "0" => "1"
    cors_rule.0.expose_headers.0:  "" => "ETag"
    cors_rule.0.max_age_seconds:   "" => "3000"
    policy:                        "" => "{\n  \"Version\": \"2012-10-17\",\n  \"Id\": \"S3StorageAnonymousRea
dPolicy\",\n  \"Statement\": [\n    {\n      \"Sid\": \"S3ReadOnly\",\n      \"Effect\": \"Allow\",\n      \"A
ction\": \"s3:GetObject\",\n      \"Resource\": \"arn:aws:s3:::staging-pfb-storage-us-east-1/*\",\n      \"Pri
ncipal\": \"*\"\n    }\n  ]\n}"
```